README.md

```diff
@@ -1 +1,6 @@
-# My-Portfolio
\ No newline at end of file
+# My-Portfolio
+This project contains my portfolio, coded according to requirements provided by Microverse.
+This project was built with HTML5, CSS3 and Linters
+Template source: Figma
+Author:
+Github: @AyomipoMajek
```

Comment:
- Please, let's improve our readme to be more professional. You can use the microverse readme [template](https://github.com/microverseinc/readme-template) and then customize it to meet the needs that suit the current project.